analytics-node/index.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Andrew Fong <https://github.com/fongandrew>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export = AnalyticsNode;
+export = AnalyticsNode.Analytics;
```

Comment:
you do not need the namespace declaration. just keep the class in the top scope and `export = Analytics`